lib/processors/lib/baseprocessor.py

```diff
@@ -8,3 +12,28 @@ class BaseProcessor(object):
     def __init__(self, *args, **kwargs):
         logging.basicConfig(format=FORMAT)
         self.logger = logging.getLogger("processor." + self.name)
+
+    def save_user_blob(self, blob, user):
+        filedata = dict(name=self.name, uid=user.userid)
+        os.makedirs("./data/{name}/user".format(**filedata), exist_ok=True)
+        if CONFIG.get('_mode') == 'dev':
+            filename = "./data/{name}/user/{uid}.pkl".format(**filedata)
+            with open(filename, 'wb+') as fd:
+                pickle.dump(blob, fd)
```

Comment:
Any reason to not json.dump rather than pickle? Just curious. Is it faster or anything?